R/simulateStochasticLogistic.R

```diff
@@ -107,8 +107,8 @@
 #'     colData = DataFrame(time = ExampleLogistic$matrix[, "time"]),
 #'     metadata = ExampleLogistic[-which(names(ExampleLogistic)=="matrix")])
 #' ExampleLogistic_SE <- mia::transformCounts(ExampleLogistic_SE, method = "relabundance")
-#' # miaViz::plotSeries(ExampleLogistic_SE, x = "time")
-#' # miaViz::plotSeries(ExampleLogistic_SE, x = "time", abund_values = "relabundance")
+#' miaViz::plotSeries(ExampleLogistic_SE, x = "time")
+#' miaViz::plotSeries(ExampleLogistic_SE, x = "time", abund_values = "relabundance")
```

Comment:
We have changed the argument name abund_values to assay_name recently, the former has been deprecated. This should be changed throughout the package. I will open an issue about this and it can be handled through a separate PR.